lib/sda/processor.go

```diff
@@ -0,0 +1,50 @@
+package sda
```

Comment:
The same thing I explained but still not implemented. I just layed down the foundations so we can do the things we talked about with 
`ProcessorHandler` and `ProcessorChannels`